src/euler3d_mappedGrid.f90

```diff
@@ -0,0 +1,679 @@
+MODULE mg
+
+  REAL(kind=8), PARAMETER :: pi=3.1415926535897932385,twopi=2*pi
+  REAL(kind=8), PARAMETER :: rEarth=637120000.0
```

Comment:
I am not sure if that's a good assumption to make, at the very least what you will get I think is the single-precision representation put into a double precision variable.  

Beyond that I am puzzled by the magnitude of the radius of the Earth here, not necessarily the exact number.  It seems like you are off by a factor of 1000?